lib/matplotlib/units.py

```diff
@@ -154,6 +154,10 @@ def get_converter(self, x):
         # DISABLED cached = self._cached.get(idx)
         # DISABLED if cached is not None: return cached
 
+        if hasattr(x, "values"):
+            # this unpacks pandas datetime objects...
```

Comment:
It is actually much more general than that; it is returning the underlying ndarray from a pandas Series or DataFrame.